xtraplatform-feature-provider-sql/src/main/java/de/ii/xtraplatform/feature/provider/sql/app/FeatureProviderSql.java

```diff
@@ -328,6 +330,46 @@ public boolean is3dSupported() {
                                                                                       }));
     }
 
+    public Optional<Period> getTemporalExtent(String typeName, String property) {
```

Comment:
What should be the return type in this case? Regular Java Period/Duration or a custom class, like BoundingBox?

Reply:
I think it should be an `Optional` of `org.threeten.extra.Interval`. 

If the minimum datetime found is null, return `Optional.empty()`. If the maximum datetime found is null, return an Interval with end set to `Instant.MAX`. This can later be queried with `Interval.isUnboundedEnd()`.